packages/dotcom-build-sass/README.md

```diff
@@ -69,4 +69,5 @@ The CSS loader has `@import` and `url()` resolution disabled as these should be
 | Option            | Type     | Default | Description                                                        |
 |-------------------|----------|---------|--------------------------------------------------------------------|
 | `webpackImporter` | Boolean  | `false` | See https://github.com/webpack-contrib/sass-loader#webpackimporter |
+| `prependData`     | String   | `''`    | See https://webpack.js.org/loaders/sass-loader/#prependdata        |
```

Comment:
The docs suggest that the attribute name should be `additionalData`, does that matter at all?

Reply:
This package has a dependency of `"sass-loader": "^8.0.0"`.

[Release 8.0.0](https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#user-content-800-2019-08-29) included:
> the `data` option was renamed to the `prependData` option

[Release 9.0.0](https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#user-content-900-2020-07-02), which was released just last Thursday, included:
> the `prependData` option was removed in favor the `additionalData` option, see [docs](https://github.com/webpack-contrib/sass-loader#additionaldata)

So while we are still using v8 we should use `prependData`, but once we update to the v9 then we should switch to `additionalData`.